bots/npm-update

```diff
@@ -99,7 +104,7 @@ def run(package, verbose=False, **kwargs):
             branch = task.branch(package, title, pathspec="package.json", **kwargs)
 
             # List of files that probably touch this package
-            lines = output("git", "grep" "{0}/".format(package))
+            lines = output("git", "grep", "{0}/".format(package))
```

Comment:
Oh, ok - that should have gone into the previous commit, I'll sort it during merging.